CONTRIBUTING.md

```diff
@@ -7,10 +7,16 @@
 
 # Prerequisites:
 
-Install VS 2019 (Professional or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
-
-**Client Libraries** are sdks used to interact with azure resources at application run time while **Management Libraries** are those used to manage (create/modify/delete) Azure resources.
-<br/><br/>
+- Install VS 2019 (Community or higher) and make sure you have the latest updates (https://www.visualstudio.com/).
+- Install the **.NET Core cross-platform development** workloads in VisualStudio
+- Install the latest version of **.NET Core SDK** for your specific platform. (https://dotnet.microsoft.com/download)
```

Comment:
I think we need to be more specific about the .NET Core SDK.  In particular we need at least .NET Core 2 preview 5 right now to be consistent to build our repo. https://dotnet.microsoft.com/download/dotnet-core/3.0.

Reply:
Do you mean .NET Core 3 preview 5?

Reply:
Yes I do mean .NET Core 3 preview 5, updated my earlier comment inline to avoid confusion.